NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs

```diff
@@ -36,6 +38,18 @@ protected override void OnCreate(Bundle savedInstanceState)
             InitLayout();
         }
 
+        protected override void OnResume()
+        {
+            base.OnResume();
+            LogUtils.LogMessage(LogSeverity.INFO, "The user is seeing Consent 3 page", null, GetCorrelationId());
+        }
+
+        protected override void OnPause()
+        {
+            base.OnPause();
+            LogUtils.LogMessage(LogSeverity.INFO, "The user is leaving Consent 3 page", null, GetCorrelationId());
```

Comment:
"Leaving" logs in OnPause is not a good solution because it is called also on page change. If you have access look at Danish app at MainApplication class at OnActivityStopped method.

Reply:
I have removed logging from the OnPause() methods, and instead mirrored the implementation from the Danish app; logging the event that the user leaves an activity in the 'OnActivityStopped' method in 'MainApplication' class.